common-libs/internal-api-clients/package-lock.json

```diff
@@ -10,6 +10,7 @@
       "license": "ISC",
       "dependencies": {
         "@affinidi/affinidi-did-auth-lib": "^1.1.0",
+        "@affinidi/tools-common": "^1.0.0-beta.1",
```

Comment:
Please do not use `npm install` in individual projects directories; use `npm run bootstrap` in the root monorepo directory instead.

There should be no dependencies on monorepo packages in `package-lock.json` (except for implicit ones brought with `affinidi-did-auth-lib`)